server/messaging/reporting.py

```
@@ -79,20 +79,32 @@ def get_notifications(db):
+            sqlQuery = """SELECT
+                            eve_MAC as MAC,
+                            eve_DateTime as Datetime,
+                            devLastIP as IP,
+                            eve_EventType as "Event Type",
+                            devName as "Device name",
+                            devComments as Comments FROM Events_Devices
+                        WHERE eve_PendingAlertEmail = 1
                             AND eve_EventType = 'New Device' {}
-                            ORDER BY eve_DateTime""".format(safe_condition)
+                        ORDER BY eve_DateTime""".format(safe_condition)
         except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)